.pre-commit-config.yaml

```diff
@@ -4,6 +4,9 @@ repos:
     hooks:
       - id: check-ast
       - id: check-json
+        exclude: invalid_json_file.json |
+              tests/test_sets/fixed_distributional_test_dataset.json |
+              tests/test_sets/expected_cli_results_default.json
```

Comment:
The first file in this list is intentionally invalid json but it's less obvious that the other 2 were meant to be.